pages/manager/PluginHandler.inc.php

```diff
@@ -68,6 +68,21 @@ function plugins($args, &$request) {
 		$templateMgr->assign('helpTopicId', 'journal.managementPages.plugins');
 
 		$site =& $request->getSite();
+		$pluginDir = dirname(dirname(dirname(__FILE__))).'/plugins/';
+		// Don't show the install link unless something can be installed somewhere
```

Comment:
I agree there should be another check when the plugin is uploaded (if there isn't already).

My concern was to hide the Install link if installation was impossible.  I think that check is reliable, no?

Can you elucidate the "safer" concern?